ccswbs/ugnext
# Fixes and refactoring

List of changes:
- Refactored Layout, Header, and Footer components to have cleaner props as well as enabling header and footer to be conditionally rendered
- Fixed header sub navigation not rendering correctly on basic pages
- Making homepage match more closely to the existing doorknob version
- Fixed icon in List component
- Added code to remove trailing slashes from NEXT_PUBLIC_DRUPAL_BASE_URL

Test Page:
- Go to https://deploy-preview-21--ugnext.netlify.app/
- Ensure homepage looks close enough to like it does on https://uoguelph.ca
- Go to https://deploy-preview-21--ugnext.netlify.app/programs/undergraduate
- Ensure program search is unaffected compared to live
